TutorLizard.Web/Views/Shared/_AdListItem.cshtml

```diff
@@ -1,30 +1,29 @@
 ﻿@model AdListItemDto
-
-<div class="card my-3">
+<div class="card my-3 ad-list-item h-100">
     <div class="card-body">
-        <h5 class="card-title">@Model.Title</h5>
+        <h5 class="card-title text-green text-center margin-bottom border-bottom">@Model.Title</h5>
         <div class="row">
-            <div class="col-6 col-sm-3">@TutorLizard.Web.Resources.Shared.Category :</div>
```

Comment:
Columns are too small for the text in almost anything but full-screen. I suggest replacing all:
- `col-sm-3` with `col-xl-5`
- `col-sm-9` with `col-xl-7`